yt/frontends/amrvac/tests/test_outputs.py

```diff
@@ -0,0 +1,104 @@
+import numpy as np # NOQA
+
+import yt # NOQA
+from yt.utilities.answer_testing.framework import \
+    requires_ds, \
+    data_dir_load, small_patch_amr
+from yt.testing import requires_file
+from yt.frontends.amrvac.api import AMRVACDataset
+
+blastwave_spherical_2D = "amrvac/bw_2d0000.dat"
+khi_cartesian_2D = "amrvac/kh_2D0000.dat"
+khi_cartesian_3D = "amrvac/kh_3D0000.dat"
+jet_cylindrical_25D = "amrvac/Jet0003.dat"
+riemann_cartesian_175D = "amrvac/R_1d0005.dat"
+
+# the following tests are not yet uploaded to yt-website ! (wip)
+#blastwave_cartesian_3D = "amrvac/bw_3d0000.dat"
+#blastwave_polar_2D = "amrvac/bw_polar_2D0000.dat"
+#blastwave_cylindrical_3D = "amrvac/bw_cylindrical_3D0000.dat"
+
+@requires_file(khi_cartesian_2D)
+def test_AMRVACDataset():
+    assert isinstance(data_dir_load(khi_cartesian_2D), AMRVACDataset)
+
+
+#@requires_ds(blastwave_cartesian_3D)
```

Comment:
For the future reference: Both `requires_ds` and `requires_file` are meant to disable a given test if the required file is missing, so there's no need to comment them out. Now that the files were uploaded to testing server, it'd have been sufficient to just re-trigger tests, if this part hadn't been commented out.

Reply:
nice, good to know !
btw, there is still one commented test that I need to get my head around because it fails to load the jet data set locally (just noticed that yesterday)